tests/users/test_dao_filter_by_skill.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+
+from flask import json
+from http import HTTPStatus
+from tests.base_test_case import BaseTestCase
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from tests.test_data import user1, user2, user3
+from tests.test_utils import get_test_request_header
+
+
+class TestFilterUsersBySkill(BaseTestCase):
+    def insert_entries_in_database(self):
+
+        # Insert data of the first entry
+        self.first_user = UserModel(
+            name=user1["name"],
+            email=user1["email"],
+            username=user1["username"],
+            password=user1["password"],
+            terms_and_conditions_checked=user1["terms_and_conditions_checked"],
+        )
```

Comment:
Dictionary unpacking should be used here
```suggestion
        self.first_user = UserModel(**user1)
```